src/components/run-button-container.js

```diff
@@ -317,7 +243,7 @@ export function RunButtonContainer({ studyUuid, currentNode, disabled }) {
                                 currentNode?.id
                             );
                         },
-                        () => setRanSensi(true),
+                        null,
```

Comment:
this will cause an error because it will try tu run null in startComputation fnThen which is not tested
```suggestion
                        () => {},
```

Reply:
done